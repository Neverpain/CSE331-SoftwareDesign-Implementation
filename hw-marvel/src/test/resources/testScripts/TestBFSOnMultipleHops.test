# Tests that the findPath method works on multistep pathing from one node to
# another, prints out path

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 e1
AddEdge g n2 n3 e2
AddEdge g n3 n4 e3

FindPath g n1 n4