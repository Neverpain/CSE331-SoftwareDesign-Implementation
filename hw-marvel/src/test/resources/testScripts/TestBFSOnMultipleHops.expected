# Tests that the findPath method works on multistep pathing from one node to
# another, prints out path

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge e1 from n1 to n2 in g
added edge e2 from n2 to n3 in g
added edge e3 from n3 to n4 in g

path from n1 to n4:
n1 to n2 via e1
n2 to n3 via e2
n3 to n4 via e3

